### ISSUE TITLE
关于 Adversarial Training Methods For Semi-Supervised Text Classification代码中的一个问题

### ISSUE THREAD

**Issue by xuanzebi** [TobiasLee/Text-Classification#15]
您好 我想问一下

       logits, self.cls_loss = cal_loss_logit(batch_embedded, self.keep_prob, reuse=False)
        embedding_perturbated = self._add_perturbation(batch_embedded, self.cls_loss)
        adv_logits, self.adv_loss = cal_loss_logit(embedding_perturbated, self.keep_prob, reuse=True)

这里为什么在不加扰动的时候，reuse=False，为什么在计算加上扰动求loss的时候reuse=True呢

谢谢~

**Comment by TobiasLee** [comment]
Hi,  因为需要重新使用之前的权重来计算这个 loss，所以 `reuse=True`

**Comment by xuanzebi** [comment]
wow  感谢回复，顿然开朗 哈哈哈
